nvm.sh

```diff
@@ -2106,19 +2106,31 @@ nvm_die_on_prefix() {
   fi
 
   local NVM_NPM_PREFIX
-  NVM_NPM_PREFIX="$(npm config --loglevel=warn get prefix)"
-  if ! (nvm_tree_contains_path "$NVM_DIR" "$NVM_NPM_PREFIX" >/dev/null 2>&1); then
-    if [ "_$NVM_DELETE_PREFIX" = "_1" ]; then
-      npm config --loglevel=warn delete prefix
-    else
-      nvm deactivate >/dev/null 2>&1
-      nvm_err "nvm is not compatible with the npm config \"prefix\" option: currently set to \"$NVM_NPM_PREFIX\""
-      if nvm_has 'npm'; then
-        nvm_err "Run \`npm config delete prefix\` or \`$NVM_COMMAND\` to unset it."
+  local npmrcs=". $HOME"
+
+  for npmrc in $npmrcs; do
+    if [ -f "${npmrc}/.npmrc" ]; then
+      NVM_NPM_PREFIX=$(grep ^prefix= ${npmrc}/.npmrc | sed s/prefix=\\\(.*\\\)$/\\1/)
+      if [[ -n $NVM_NPM_PREFIX ]]; then
+        break
+      fi
+    fi
+  done
+
+  if [[ -n $NVM_NPM_PREFIX ]]; then
```

Comment:
double brackets are bash-only syntax; it's not in POSIX.

Reply:
got it looking at it